### PR TITLE
[add] Implementation for SERVICETALK: json/ plaintext

### PR DESCRIPTION
**Pull request by arthur-techempower**

### CODE REVIEW

File: frameworks/Java/servicetalk/src/main/java/com/techempower/servicetalk/Server.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    public static void main(String[] args) throws Exception {
+        try{
+            HttpSerializationProvider serializer = HttpSerializationProviders.jsonSerializer(new JacksonSerializationProvider());
+            Map<String, String> obj = new HashMap<String, String>();
+            obj.put("message", "Hello, World!");
+            HttpServers.forPort(8080)
+                    .listenAndAwait((ctx, request, responseFactory) -> {
+                        if(request.path().equals("/json")) {
+                            return succeeded(responseFactory.ok()
+                                    .payloadBody(obj, serializer.serializerFor(Map.class))
```
Comment (zloster) [inline frameworks/Java/servicetalk/src/main/java/com/techempower/servicetalk/Server.java]: Here new object must be instantiated. See the first rule (`i`): https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#requirements

File: frameworks/Java/servicetalk/pom.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <finalName>${project.artifactId}</finalName>
+        <plugins>
+            <plugin>
+                <inherited>true</inherited>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
```
Comment (zloster) [inline frameworks/Java/servicetalk/pom.xml]: Most of the frameworks are already on Java 11 without using modules i.e. there are warnings when the JVM starts. Is there any reason not to use it?

**Comment by zloster** [comment]
@nbrady-techempower The move of `Java/sabina` in the `.travis.yml` is lost in the new PR.

**Comment by nbrady-techempower** [comment]
thanks @zloster I'd actually like to start combining similar Java tests to save time on the runs

**Pull request closed without merging**